releasenotes/notes/cassandra-latest-3-79c9f21c080bd1c6.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Added support for the latest Cassandra release.
```

Comment:
Don't think it's worth the release note as we already mention in our docs that we support cassandra 3.5 and above - maybe makes sense as a `fix` (and maybe mention which specific versions we fixed)

```suggestion
```